src/wasm.h

```diff
@@ -1444,12 +1444,13 @@ class Import {
     Global = 3,
   };
 
-  Import() : functionType(nullptr), globalType(none) {}
+  Import() : functionType(nullptr), globalType(none), globalMutable(false) {}
 
   Name name, module, base; // name = module.base
   Kind kind;
   FunctionType* functionType; // for Function imports
   WasmType globalType; // for Global imports
+  bool globalMutable; // for Global imports
```

Comment:
My understanding is that mutable globals can't be imported, as verified by the s-expression tests. So we don't need this field, unless it's for future use? If it is just for future use, then maybe this could wait for future refactoring of wasm.h (in particular I'd like to get rid of the Import class entirely, and then a Global could have an imported flag, and Global already has a mutable field).